bin/server/cmd/profile.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/bytebase/bytebase/server"
 )
 
-func getBaseProfile() server.Profile {
+func getBaseProfile(backupMeta backupMeta) server.Profile {
```

Comment:
Can we part the flag here so that we don't need backupMeta struct? There's no need for getBaseProfile to take an extra parameter.